tracing/tracing/src/main/java/module-info.java

```diff
@@ -40,5 +41,6 @@
     exports io.helidon.tracing.spi;
 
     uses io.helidon.tracing.spi.TracerProvider;
+    uses SpanListener;
```

Comment:
Please use fully qualified class name.

Reply:
Fixed.

Reply:
I still see it without fully qualified class name.